src/components/InputWithLabel.jsx

```diff
@@ -0,0 +1,30 @@
+import "./inputWithLabel.css"
+function InputWithLabel(props) {
+    const type = props.Type;
```

Comment:
we may use object destructureing here for cleaner code and it would help if we pass other props.